### PR TITLE
[spike]: Link to /cart

### PR DESCRIPTION
**Pull request by sirugh**
## Description
With 7.0.0 coming up I wanted to demonstrate the workflow for a user adding an item to their cart and then getting to the cart page. Whether we include this solution or some semblance of it in 7.0.0 is debatable but the PR can at least be used as an example or referenced in documentation.

Changes:
* When you click "Checkout" in the mini cart it now navigates to the `/cart` page.
* On the `/cart` page the mini cart trigger no longer renders.

## Related Issue

Closes #ISSUE_NUMBER.

### Verification Steps

1. Add a product to your cart.
2. In the mini cart, click the "Checkout" button. See that you are navigated to /cart.
3. On the /cart page, see that the trigger is not rendered.

We should also ensure that adding/removing items through the PDP or the mini cart update the /cart page accordingly.

## Screenshots / Screen Captures (if appropriate)

[Image from Gyazo](https://gyazo.com/1df02abb3f164959cb6ce4ba7ef66dad)

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: packages/venia-ui/lib/components/Checkout/flow.js [hunk 1]
```diff
@@ -72,7 +73,7 @@ const Flow = props => {
                 <div className={classes.footer}>
                     <CheckoutButton
                         disabled={checkoutDisabled}
-                        onClick={handleBeginCheckout}
+                        onClick={handleGoToCart}
```
Comment (sirugh) [inline packages/venia-ui/lib/components/Checkout/flow.js:76]: If a developer is upgrading from 6.0.0 to 7.0.0 and doesn't want to use the new cart/checkout experience they can simply set this back to `handleBeginCheckout`.

**Comment by devops-pwa-codebuild** [comment]
**Performance Test Results**

The following fails have been reported by WebpageTest. These numbers indicates a possible performance issue with the PR which requires further manual testing to validate.

https://pr-2475.pwa-venia.com : LH Performance Expected 0.85 Actual 0.52, LH Best Practices Expected 1 Actual 0.92
https://pr-2475.pwa-venia.com/venia-tops.html : LH Performance Expected 0.75 Actual 0.35, LH Best Practices Expected 1 Actual 0.92
https://pr-2475.pwa-venia.com/valeria-two-layer-tank.html : LH Performance Expected 0.8 Actual 0.44, LH Accessibility Expected 0.9 Actual 0.89, LH Best Practices Expected 1 Actual 0.92

**Comment by fooman** [comment]
Shouldn't it navigate to `/checkout` instead? Seems odd to click "Checkout" and then having to click "Proceed to Checkout" once more.

**Comment by sirugh** [comment]
> Shouldn't it navigate to /checkout instead? Seems odd to click "Checkout" and then having to click "Proceed to Checkout" once more.

Maybe. I'd have to check the mocks for the new mini cart work but the idea is that the button in the mini cart links to the /cart page and from there you go to /checkout when you're happy with your cart.

The button text is certainly debatable :)

**Comment by sirugh** [comment]
I think this is going to get done in some fashion. I'll drop a link to this spike/PR in https://jira.corp.magento.com/browse/PWA-737

**Pull request closed without merging**